### PR TITLE
inventory interact inventoryToEphemeralTransferWithParam with paramater

### PR DESCRIPTION
**Pull request by 0xxlegolas**
bug fix - inventory interact inventoryToEphemeralTransferWithParam with paramater

### CODE REVIEW

File: mud-contracts/world/src/modules/inventory/systems/InventoryInteract.sol [hunk 1]
```diff
@@ -84,6 +84,64 @@ contract InventoryInteract is EveSystem {
     _inventoryLib().depositToEphemeralInventory(smartObjectId, ephemeralInventoryOwner, inItems);
   }
 
+  /**
+   * @notice Transfer items from ephemeral to inventory
+   * @dev transfer items from ephemeral to inventory
```
Comment (fc-tezza) [inline mud-contracts/world/src/modules/inventory/systems/InventoryInteract.sol]: These comments are incorrect, this is a transfer from inventory to ephemeral

File: mud-contracts/world/src/modules/inventory/systems/InventoryInteract.sol [hunk 2]
```diff
@@ -84,6 +84,64 @@ contract InventoryInteract is EveSystem {
...
+   * @param smartObjectId is the smart object id
+   * @param ephemeralInventoryOwner is the ephemeral inventory owner
+   * @param outItems is the array of items to transfer
+   */
+  function inventoryToEphemeralTransferWithParam(
+    uint256 smartObjectId,
+    address ephemeralInventoryOwner,
```
Comment (ccp-tezza) [inline mud-contracts/world/src/modules/inventory/systems/InventoryInteract.sol:96]: For interfacing this is clearer, but technically we can just use the parameter in the InventoryItem object (as now it is not being used). In any case, I am fine to leave it as is for now.

**Review by fc-tezza — APPROVED** [review]
Just a few sideline comments, but overall looks good to me.

File: mud-contracts/world/src/modules/inventory/systems/InventoryInteract.sol (hunk 1 above)
Comment (0xxlegolas) [reply mud-contracts/world/src/modules/inventory/systems/InventoryInteract.sol]: fixed

**Pull request merged**